content/blog/entries/cc-vocabulary-docs-updates-2/contents.lr

```diff
@@ -0,0 +1,87 @@
+
+<center>
+<img alt"Halfway There" src="speed.gif"/><br>
+<small class="muted">Hitting "the point of no return" has never been this exciting! Time to step on the throttle! Source: <a href="https://cliply.co">Cliply</a></small>
+</center>
+
+### What I've been up to
+
+> **Designing** ⇨ **Drafting** ⇨ **Developing** ⇨ **Debugging** ⇨ **Deploying**
+
+And the cycle contrinues. I guess it sums it all up very nicely. _Can somebody appreciate the alliteration though?_ 
+
+Here's a gist of what I've achieved so far:
+
+- I've gone through **2** iterations of the design. I'm happy with how the new site looks (and I genuinely hope the design team does too!).
+- I've drafted around **5+** writeups dealing with Monorepo Migration, Getting Started guide, Vocabulary Overview and of course these blog posts.
+- My branch on the vocabulary repository now has over **50+** commits & over **13,000** lines of code (not that I've written all of them, but you know, just for the stats)
+- The first draft of the vocabulary site is now live! I'm expecting a whole bunch of changes still, but here it is if you want to have a sneak peek: [https://cc-vocab-draft.surge.sh](https://cc-vocab-draft.surge.sh)
+-  I've consumed the [Github API](https://docs.github.com/en/free-pro-team@latest/rest) to get live release history, forks and starrers count. I think it adds a really nice touch to the site in general.
+- I've used [surge.sh](https://surge.sh) to deploy the draft site. I believe it's a really simple tool to have your site deployed within seconds!
+
+<center>
+<img alt"Github commit gif" src="github.png"/><br>
+<small class="muted">My github contribution chart is filling up!</small>
+</center>
+
+### What I've learnt
+
+Some say it's hard to learn through virtual internships. Well, let me prove you wrong. Here are my leanings in the past few weeks:
+
+- It's surprising how subjective (& yet objective) designing really is.
+- Vue.js is _fantastic_. Maybe I'm a VueJS fan now. Should I remain loyal to React? I don't know.
+- Making a site responsive isn't the _easiest_ of tasks, but it's certainly doable after a lot of stretching & compressing; lets say that.
+- "Code formatting is essential" would be an _understatement_ to make.
+- Monorepo's have their own pro's and con's. But in our case the con's were negligible, thankfully! 
+-  I'll be following up with some performance & accessibility testing this coming week, so let's see how that plays out!
+- A mentor plays a vital role in any project. My mentor `@dhruvkb` has been very supportive and has made sure I stick to my timeline!
+
+### Other community work tidbits
+
+I believe apart from the internship work that I'm engaged in, I should also help around with some community PR work. I've been told I'm always welcome to, which is great!
+
+- I got the opportunity to speak at a CCOS event alongwith fellow speakers `@dhruvkb` & `@dhruvi`. I had a blast talking to budding students from DSC-IIT Surat & DSC-RIT.
```

Comment:
Why the `@`?

Reply:
I thought the slack username would make more sense here. But in hindsight, I think I'll just link to your authors page 😄